build.sh

```diff
@@ -6,8 +6,7 @@ mkdir -p build
 cd build
 
 # Run cmake to generate make files
-cmake ..
+cmake -DCMAKE_BUILD_TYPE="Release" ..
```

Comment:
I presume that's way to complicated for people, unfortunately. Also not sure why it's not the default

Reply:
Confused by what you mean here.

Reply:
For some reason I thought this was README. But on that note it would be weird to have build sh do sth special not added to README

Reply:
I don't think it would really do something "special" per se. Running build.sh would just automatically generate a release build - which is pretty much what it does now. I think it makes much more sense for _debug_ builds to be opt-in.

Reply:
Yes, debug should be opt-in so the Q is why "cmake .." is not sufficient for a release build.

Reply:
The CMAKE_BUILD_TYPE flag isn't being set to "Release" for single-config generators, so no release flags are being applied if you just run cmake ..

This isn't a problem for multi-config generators, which will ignore the CMAKE_BUILD_TYPE flag anyway.

I _guess_ I could try and jerry-rig a system where if a configuration option isn't passed, CMAKE_BUILD_TYPE is set to release by default. Is that really desired?